src/screens/Splash/Splash.js

```diff
@@ -1,11 +1,30 @@
-import React from "react";
+import React, { useEffect } from "react";
 import { View, Image } from "react-native";
+import AsyncStorage from "@react-native-community/async-storage";
 
 import styles from "./styles";
 
-const splashImg = require("assets/splash/Splash.png");
+const splashImg = require("../../assets/splash/splash.png");
+
+const getInitialRoute = async () => {
+  let initialRoute;
+  const isShowIntroduction = await AsyncStorage.getItem("isShowIntroduction");
+  if (isShowIntroduction !== "false") {
```

Comment:
if (isShowIntroduction) {

Reply:
AsyncStorage return only string. I think convert sctring to boolean is not nessesary here

Reply:
agree